src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/VSCodeAnalyzerLoader.cs

```diff
@@ -25,21 +26,21 @@ public void InitializeDiagnosticsServices()
     {
     }
 
-    public static IAnalyzerAssemblyLoader CreateAnalyzerAssemblyLoader(ExtensionAssemblyManager extensionAssemblyManager, ILogger logger)
+    public static IAnalyzerAssemblyLoader CreateAnalyzerAssemblyLoader(HostWorkspaceServices services, ExtensionAssemblyManager extensionAssemblyManager, ILogger logger)
```

Comment:
Should we switch this to be an export of the workspace service now?

Reply:
we need the wrapped one that knows how to load assemblies from the extension context.  But we only do that for solution level analyzers, so we need two different versions.  This would have to be a different workspace service so it doesn't get used when we're loading project level analyzers.

Reply:
Can we just unify to one though? Since we already have the TryLoadAssemblyInExtensionContext... method, wouldn't this work for the other case too?

Reply:
done